<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminWindow</class>
 <widget class="QMainWindow" name="AdminWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>781</width>
    <height>693</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0" rowspan="4">
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>18</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Bank Data Base</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableWidget" name="tbl_view_database">
        <column>
         <property name="text">
          <string>AccountNumber</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Username</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Name</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Balance</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Age</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="pbn_view_database">
          <property name="text">
           <string>View Database</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lbl_view_database_error">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="4" column="0" rowspan="2">
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>18</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Transaction History</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableWidget" name="tbl_transaction_history">
        <column>
         <property name="text">
          <string>Amount</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Date</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Time</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="lbl_err_transaction_history">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_14">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Account Number :</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lnedit_act_number_transaction_history"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_10">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Counts :                  </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lnedit_num_of_counts_trans_history"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="pbn_view_transaction_history">
          <property name="text">
           <string>View Transaction history</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="5" column="1" colspan="2">
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QLabel" name="lbl_accountdeletion">
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>18</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Delete Account</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="lbl_account_number_delete">
          <property name="text">
           <string>Account Number:   </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lnedit_delete_account_number"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QPushButton" name="pbn_delete_account">
          <property name="text">
           <string>Delete Account</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lbl_error_delete">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="0" column="2">
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>18</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Account Number</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_view_account_number">
        <property name="text">
         <string>Account Number: </string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_12">
        <item>
         <widget class="QLabel" name="label_username">
          <property name="text">
           <string>username:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_username"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QPushButton" name="pushButton_get_account_number">
          <property name="text">
           <string>Get Account Number</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_error">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>18</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>View Account Balance</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lbl_viewbalance">
          <property name="text">
           <string>Balance:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_error_viewbalance">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_11">
        <item>
         <widget class="QLabel" name="lbl_accountnumber">
          <property name="text">
           <string>Account Number: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lnedit_accountnumber"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="pbn_view_balance">
        <property name="text">
         <string>View Balance</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="4" column="2">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="lbl_accountcreation">
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>18</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Create / Update Account</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="lbl_username_input">
          <property name="text">
           <string>Username: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lnedit_username_input"/>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="lbl_password">
          <property name="text">
           <string>Password:  </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lnedit_password_input">
          <property name="echoMode">
           <enum>QLineEdit::PasswordEchoOnEdit</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="lbl_name">
          <property name="text">
           <string>Name:        </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lnedit_name_input"/>
        </item>
       </layout>
      </item>
      <item row="4" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="lbl_age">
          <property name="text">
           <string>Age:            </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lnedit_age_input"/>
        </item>
       </layout>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="lbl_error_create">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QPushButton" name="pbn_create_new_account">
          <property name="text">
           <string>Create New Account</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="chkbox_admin">
          <property name="text">
           <string>Admin</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="7" column="0">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_15">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Account Number:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lnedit_update_acc_num"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="pbn_update_account">
          <property name="text">
           <string>Update Account</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>781</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
